demo/src/main/java/com/orange/ods/demo/ui/components/cards/ComponentCard.kt

```diff
@@ -42,12 +45,25 @@ fun ComponentCard(variant: Variant) {
                         text = stringResource(id = R.string.component_element_thumbnail),
                         trailing = OdsSwitchTrailing(checked = thumbnailChecked)
                     )
+                } else if (variant == Variant.CardHorizontal) {
+                    OdsChoiceChipsFlowRow(
```

Comment:
We could display a title saying `Image position` to explain which property is customized by the chips.